src/main/resources/entity-types/configuration/drv_config_data_details.json5

```diff
@@ -0,0 +1,170 @@
+{
+  id: 'abb28bac-e090-48c1-a48b-3fe805638041',
+  name: 'drv_config_data_details',
+  private: false,
+  fromClause: 'src_configuration_config_data as config',
+  columns: [
+    {
+      name: 'id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      isIdColumn: true,
+      valueGetter: "config.id"
+    },
+    {
+      name: 'module',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "config.jsonb ->> 'module'"
+    },
+    {
+      name: 'config_name',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "config.jsonb ->> 'configName'"
+    },
+    {
+      name: 'code',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "config.jsonb ->> 'code'"
+    },
+    {
+      name: 'description',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "config.jsonb ->> 'description'"
+    },
+    {
+      name: 'default',
+      dataType: {
+        dataType: 'booleanType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "config.jsonb ->> 'default'",
+      values: [
+        {
+          label: 'True',
+          value: 'true',
+        },
+        {
+          label: 'False',
+          value: 'false',
+        },
+      ]
+    },
+    {
+      name: 'enabled',
+      dataType: {
+        dataType: 'booleanType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "config.jsonb ->> 'enabled'",
+      values: [
+        {
+          label: 'True',
+          value: 'true',
+        },
+        {
+          label: 'False',
+          value: 'false',
+        },
+      ]
+    },
+    {
+      name: 'value',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "config.jsonb ->> 'value'"
+    },
+    {
+      name: 'user_id',
+      dataType: {
+        dataType: 'stringType',
```

Comment:
this should probably be rangedUUIDType (really it should be openUUIDType but we're not using that atm)

Reply:
@bvsharp  done